src/environment.jl

```diff
@@ -1,25 +1,26 @@
 
-Base.@kwdef struct EnvironmentalParams{A,P,E,L,F}
+Base.@kwdef mutable struct EnvironmentalParams{A,P,E,L,F}
```

Comment:
Do we need for mutability? I normally try to avoid it, as it's slower and can't be used with GPUs.

Reply:
Not totally sure - when we're allowing behavioural changes, we'd want the option for changing the environment and some traits. Without behaviour we wouldn't need it.